### ISSUE TITLE
Dial click no longer working after update to Stream Deck software v6.6.0

### ISSUE THREAD

**Issue by adhawkins** [GeekyEggo/SoundDeck#132]
It looks like the way dial click events are notified has changed in the new Stream Deck software v6.6.0.

As a result, a click on the dial for a 'Set Audio Device Volume' button on the Stream Deck + no longer works (it should toggle mute).

**Comment by GeekyEggo** [comment]
Hi @adhawkins, just to confirm, what version of Sound Deck are you using?

**Comment by adhawkins** [comment]
3.10.1

**Comment by GeekyEggo** [comment]
Thank you, I'll add this to the backlog; a workaround until there is a fix is to use the touchscreen tap. 😄

**Comment by adhawkins** [comment]
Yeah, I was informed of the workaround while reporting a similar issue with another plugin. Can confirm that this works fine.
